prisma/seed.ts

```diff
@@ -54,9 +60,16 @@ const venueSchema = {
   "name": { prop: "name", type: String, required: true }
 };
 
+type UserOnOrgSchemaType = {
+  userId: number,
+  orgId: number,
+  isIGHead: number
```

Comment:
hmm why is this a number?

Reply:
It can be a string with 'true' and 'false' also, but 0 and 1 was easier to type and fill up - I can add better validation for this